src/components/general/ProgressBar.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import LinearProgress from "@material-ui/core/LinearProgress";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    width: "100%",
+    "& > * + *": {
+      marginTop: theme.spacing(2)
+    },
+    textAlign: "center"
+  }
+}));
+
+export default function ProgressBar(props) {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      {props.progressPercentage}%
```

Comment:
Should this be rounded? For example, what if someone passes a percentage of `1.23`? Should that render as `1.23%`, or `1%`?